maintenance/troubleshoot/vpp/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: VPP dataplane
+description: Architecture, configuration reference & troubleshooting guide for the VPP dataplane
```

Comment:
Architecture, configuration reference, and  troubleshooting for the VPP dataplane.